libpod/runtime_ctr.go

```diff
@@ -393,6 +402,14 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool)
 		}
 	}
 
+	for _, v := range volumes {
+		if volume, err := runtime.state.Volume(v); err == nil {
+			if err := runtime.removeVolume(ctx, volume, true, true); err != nil {
```

Comment:
If we get an error that the volume is still in use, we should log nothing

Reply:
Fixed to ignore error if I get back the errnovolume or errcontainerbeingused